src/pages/Graph/GraphRouteHandler.tsx

```diff
@@ -132,9 +108,11 @@ export class GraphRouteHandler extends React.Component<RouteComponentProps<Graph
     // to use the redux namespace
     // @ts-ignore
     const nextNamespace = props.activeNamespace;
+    // @ts-ignore
+    const reduxDuration = props.duration; // get from Redux
+
     const nextNode = GraphRouteHandler.getNodeParamsFromProps(props);
     const {
-      graphDuration: nextDuration,
```

Comment:
I'm worried that this will lead to bookmark-ability issues, since this is the only place where I see the duration was being fetched from the URL. Without this, looks like redux will always override whatever is in the URL.

Reply:
Redux will override because it needs to since another part (page) of the application has set it and we want to use the global redux setting.

Reply:
If we are comfortable with overriding, my suggestion would be to remove the duration from the URL parameters. That way, bookmark-ability it won't be "broken" and I see this quite minor, since it's an URL parameter that is not "too important" when compared to the namespace.

Reply:
(although, I really would like to keep the parametert for bookmarking)

Reply:
We could then agree on a strategy where:

1.  if there are no url params then we use redux and
2. if there are url params we use that value and set the redux value to that

However, think this should get merged and we create a Jira for this strategy

Reply:
As mentioned above, I think we do need to keep duration in the URL and I think we can do it in the same way namespaces is handled.  I think we don't want a GraphRouteHandlerContainer.  We need to talk about it.